### ISSUE TITLE
link fixes

### ISSUE THREAD

**Issue by architdate** [bumbadadabum/Kid-A#21]
It seems like you are not parsing video links arguments like timestamp via youtube (`?t=`). Because of this whenever someone uses the .link command with a timestamped youtube url, the `getVideoIdFromURL` function just takes a substring after `youtu.be/` or `?v=` which may include other arguments such as timestamp. This leads to invalid IDs and so Kid A thinks that its an invalid video. 

A suggested solution would be that once you calculate your ID your normal way, do something like: `id = id.split('?')[0]` to get the valid id without any arguments in them. You won't be able to do this directly to the passed url argument because that would truncate the non shortened youtube links which need the ?v= to work. I can open a pr to fix this, but since there is no CODE-OF-CONDUCT.md file to follow, I thought it might be better to leave this to you. Let me know if you require any further clarification

**Comment by Asheviere** [comment]
yeah that shouldn't be too hard to fix. Don't know how soon I'll be able to get to it, but yeah that's a good thing, never considered this was an edgecase.
